src/Tasks/Microsoft.NET.Build.Tasks/ProcessFrameworkReferences.cs

```diff
@@ -609,6 +614,36 @@ private bool AddCrossgen2Package(Version normalizedTargetFrameworkVersion, List<
             }
         }
 
+        private string GetPackPath(string packName, string packVersion)
+        {
+            IEnumerable<string> GetPackFolders()
+            {
+                if (!string.IsNullOrEmpty(TargetingPackRoot))
+                {
+                    yield return TargetingPackRoot;
+                }
+                var packRootEnvironmentVariable = Environment.GetEnvironmentVariable("DOTNETSDK_WORKLOAD_PACK_ROOTS");
```

Comment:
Do we have a central class where we can keep all these environmental variable constants? That way we don't need to worry about typo's in the code.

Reply:
Thank you. I think this will save us a lot of potential pain in the future

Reply:
I've added one now (`EnvironmentVariableNames`) and added constants for the workload environment variables.  There are a bunch of other CLI environment variables which I didn't refactor into this class yet.